tests/phpunit/api/v4/Action/GetFromArrayTest.php

```diff
@@ -122,6 +122,8 @@ public function testArrayGetWithWhere(): void {
       ->execute();
     $this->assertEquals([4, 5], array_column((array) $result, 'field1'));
 
+    // TODO - Add REGEXP BINARY and NOT REGEXP BINARY tests here, maybe
```

Comment:
I see how this can be confusing, because the test doesn't save any data and then reads it back from... somewhere.
I had to look in the MockArrayEntity definition to realize that it's literally just a static array of data that never changes (essentially a read-only entity). Here is what it reads from when you call `MockArrayEntity::get()`:
https://github.com/civicrm/civicrm-core/blob/f452d72c08c27b1881af42f12c669185c43c3e30/tests/phpunit/api/v4/Mock/Api4/Action/MockArrayEntity/Get.php#L28-L67

Reply:
It would be good as part of this PR to update that dataset with an uppercase letter somewhere so we can have a meaningful test.

Reply:
Yes that's the bit I needed to know.